setup.cfg

```diff
@@ -7,7 +7,7 @@ all_files = 1
 upload-dir = docs/_build/html
 show-response = 1
 
-[pytest]
+[tool:pytest]
```

Comment:
this won't work for older pytest, resulting the skip of this config section and potential weird behaviour (e.g https://github.com/astropy/astropy/issues/5695), so if you update it, then it's better to update minversion to 3.0 below, too.